spec/integration/application_plan_limit_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require_relative '../shared_tests_config'
 
-RSpec.describe 'Application plan limit resource', type: :integration do
+describe 'Application plan limit resource', type: :integration do
```

Comment:
This is against RSpec best practices. It is not recommended to pollute global namespace. Default rspec installation will forbid this. I'd recommend using the RSpec namespace.